src/utils/findGitmojiCommand.js

```diff
@@ -1,16 +1,26 @@
 // @flow
-import COMMIT_MODES from '@constants/commit'
+import COMMIT_MODES, { COMMIT_MESSAGE_PARTS } from '@constants/commit'
 import FLAGS from '@constants/flags'
 
 const getOptionsForCommand = (command: ?string, flags: Object): ?Object => {
   const commandsWithOptions = [FLAGS.COMMIT, FLAGS.HOOK]
 
   if (commandsWithOptions.includes(command)) {
+    const gitFlags = omit(
+      flags,
+      'm',
+      'add',
+      ...Object.values(COMMIT_MESSAGE_PARTS),
+      ...Object.values(FLAGS)
+    )
+    // TODO(anau) check if message, mode, scope, title in git flags (-m ":mode: (scope) title" -m "message" -> 'm' is array if 2 times) includes #1037
```

Comment:
here the check for #1037 is possible to be included.
Additional  it makes sense to add a check if `gitmoji` is called e.g. with
`gitmoji -c -m ":tada: (bootstrap) bootstrapping project"` (maybe in that case it does not make sense to use gitmoji, but possible with hook, or automation)